packages/ui/src/elements/DocumentHeader/Tabs/Tab/TabLink.tsx

```diff
@@ -25,6 +25,10 @@ export const DocumentTabLink: React.FC<{
 }) => {
   const pathname = usePathname()
   const params = useParams()
+
+  const searchParams = useSearchParams()
+  const locale = searchParams.get('locale')
```

Comment:
You will want to pull this from the `useLocale` hook.

Reply:
As discussed, there may be situations where the locale isn't URL enforced, so using the URL params keeps it more consistent than always appending a locale